pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -137,7 +138,15 @@ func Execute(configFile io.Reader) {
 	app.Config.HTTP.Headers.Set("X-Registry-Supports-Signatures", "1")
 
 	app.RegisterHealthChecks()
-	handler := alive("/", app)
+	handler := http.Handler(app)
```

Comment:
The limiter is fine, but this needs to be much more discriminating.  Blob HEAD and GET shouldn't be under the same rate limit. This is too broad to solve the existing problem without adding a new one.  

This needs to only apply to blob upload.